modules/network-observability-flowcollector-about.adoc

```diff
@@ -0,0 +1,116 @@
+<3> The Loki specification, `spec.loki` configures how to reach Loki. The default values match the Loki install paths mentioned in the Installing the Loki Operator section, but you may have to configure differently if you used another installation method.
+
+= Flow Collector Configuration Parameters
+The fields for the Flow Collector custom resource are described in the following table.
+
+.FlowCollector custom resource configuration parameters
+[cols="1,1,8a",options="header"]
+|===
+
+|Field
+|Type
+|Description
+
+|`spec.agent.type`
+|`string`
+|Specifies the agent for flows extraction. eBPF is the default agent and is the only {product-title} supported option; IPFIX is a functional option but it is unsupported.
+
+|`spec.agent.ebpf`
+|`integer`
+|Specifies the the settings related to the eBPF-based flow reporter. The following eBPF settings are helpful for managing compute resources when continuously sampled flow collection might consume a large amount of resources:
+
+* `ebpf.sampling` specifies the sampling rate of the flow reporter. A value of 100 means one flow every 100 is sampled. A value of 1 means all flows are sampled. The lower the value, the increase in returned flows and the accuracy of derived metrics. The default value is 50. 
+* `ebpf.resources`specifies the compute resources required by the container. The default value is 800 MB, which is sufficient for most medium-sized clusters
+* `ebpf.cacheMaxFlows` and `ebpf.cacheActiveTimeout` specify how often flows are reported by the agents. The higher the `cacheMaxFlows` and `cacheActiveTimeout` values, the less traffic the eBPF agent generates, which reduces resource demands; however note that memory consumption is higher which may create latency in the flow collection.
+* `ebpf.interfaces` and `ebpf.excludeInterfaces` specify the interface names from where flows are collected. You can reduce the overall observed traffic by restricting or excluding interfaces, which can help with resrouce management. If empty, the agent will fetch all the interfaces in the system, excepting the ones listed in ExcludeInterfaces. If an entry is enclosed by slashes (e.g. `/br-/`), it will match as regular expression, otherwise it will be matched as a case-sensitive string.
+* `ebpf.env` specifies custom environment variables to the
```

Comment:
This has been moved to `ebpf.debug.env`. @jotak do we want to expose this as documented? In my opinion, it should.

Reply:
Agreed that it should be documented

Reply:
I'm fine also with that ; just need to fix the path as Mario says

Reply:
Addressed